more/pixels/pixelMintController.js

```diff
@@ -12,40 +13,45 @@ export function makeMintController(assay) {
   // pixel to purse/payment
   const pixelToPursePayment = new Map();
 
-  function setLocation(amount, pursePayment) {
-    // purse/payment is the key of rights
+  function setPixelsToAsset(amount, purseOrPayment) {
```

Comment:
I like purseOrPayment, but it's so long that I was afraid of getting called out for long names :). What do you think of Asset?

Reply:
I don't mind 'Asset', as long as we're consistent. But now with the decision to enforce the distinction between Purses and Payments, perhaps we won't need a new word for both?

Reply:
There are still arguments that are either a purse or payment, for instance in `takePayment`, so I think we still have this problem even with `rights` being split. I just pushed up a commit that uses 'asset' instead of 'purseOrPayment'. Let me know what you think!